### PR TITLE
Add documentation for the release process

### LINKED ISSUE #80: Add documentation for the release workflow
## Purpose

We need to document our workflow to make a new Pixel's release.

## Proposal

Write this documentation!

### PR DESCRIPTION
**Pull request by willdurand**
Fix #80

### CODE REVIEW

File: docs/release-process.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+   `VERSION`, `pixel/__init__.py` and `package.json`
+2. update the `CHANGELOG` file with a description of the new features, bug
+   fixes, and so on for the end users
+3. commit the version changes as well as the updated `CHANGELOG` file (_e.g._,
+   message could be `Prepare 1.1.0 release`)
+4. push to `master` on GitHub and wait for the CI status. The commit to prepare
+   the release MUST pass (_i.e._ be "green"). Alternatively, you can create a
+   branch and open a pull request that will get merged after the step 7
+5. tag the version locally (_e.g._, `git tag 1.1.0`)
+6. push the tag to GitHub by using the `--tag` option on `git push`
```
Comment (jmaupetit) [inline docs/release-process.md]: Should be `--tags` with a `s`.

File: docs/release-process.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+7. go to https://github.com/Candihub/pixel/releases and create a GitHub release
+   from the git tag you've just pushed. Give a name to the release (at least the
+   version number) and (ideally) copy/paste the content you've written in the
+   `CHANGELOG`
+
+Note: a _release_ is another name to refer to a _stable version_, because we
+"release" stable versions only. All "releases" (or "stable versions") are tagged
+in git, _i.e._ (git) tags refer to releases.
```
Comment (jmaupetit) [inline docs/release-process.md]: When talking about the software, should we write `git` (the command) or Git or git?

File: docs/release-process.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+1. update the version number of the project by updating the following files:
+   `VERSION`, `pixel/__init__.py` and `package.json`
+2. update the `CHANGELOG` file with a description of the new features, bug
+   fixes, and so on for the end users
+3. commit the version changes as well as the updated `CHANGELOG` file (_e.g._,
+   message could be `Prepare 1.1.0 release`)
+4. push to `master` on GitHub and wait for the CI status. The commit to prepare
+   the release MUST pass (_i.e._ be "green"). Alternatively, you can create a
+   branch and open a pull request that will get merged after the step 7
```
Comment (jmaupetit) [inline docs/release-process.md]: As the `master` branch is protected, I think we need to prefer and only document the second strategy (PR).

Comment (jmaupetit) [reply docs/release-process.md]: Yup. You and I are allowed to do so 😬 but we should document the expected process.

File: docs/release-process.md (hunk 1 above)
Comment (willdurand) [reply docs/release-process.md]: both work :p

**Comment by willdurand** [comment]
updated!

**Pull request merged**